blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractPipelineCreateRequest.java

```diff
@@ -72,4 +72,8 @@ protected ModifiableTopLevelItemGroup getParent() {
 
         return parent;
     }
+
+    protected BlueOrganization findOrganization() {
+        return OrganizationFactory.getInstance().get(getOrganization());
```

Comment:
@vivek agreed. I think we can address this when removing the "get first organization" from the preloaders problem.